tests/test_models/test_linear_model.py

```diff
@@ -52,6 +52,19 @@ def linear_segments_ts_common():
     return linear_segments_by_parameters(alpha_values, intercept_values)
 
 
+@pytest.mark.parametrize("model", (LinearPerSegmentModel(), ElasticPerSegmentModel()))
+def test_not_fitted(model, linear_segments_ts_unique):
+    """Check exception when trying to forecast with unfitted model."""
+    train, test = linear_segments_ts_unique
+    lags = LagTransform(in_column="target", lags=[3, 4, 5])
+    train.fit_transform([lags])
+    test.fit_transform([lags])
```

Comment:
it's not important here but usually you don't need to use transforms with test dataset 🤔